security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -43,9 +43,9 @@ const (
 	// SecretType is used for secret discovery service to construct response.
 	SecretType = "type.googleapis.com/envoy.api.v2.Secret"
 
-	// credentialTokenHeaderKey is the header key in gPRC header which is used to
+	// CredentialTokenHeaderKey is the header key in gPRC header which is used to
 	// pass credential token from envoy to SDS.
```

Comment:
"from Envoy to SDS" is imprecise. from Envoy's SDS request.

Reply:
updated.